cloudformation_templates/aws_digitalmarketplace_supplier_frontend.json

```diff
@@ -1,3 +1,28 @@
 {% extends "app_base.j2" %}
```

Comment:
This doesn't look like a `.json` file - should this also be `.j2`?  Or is Cloudformation happy with this as JSON?